wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/ast2wdlom/AstToRuntimeAttributeElement.scala

```diff
@@ -0,0 +1,18 @@
+package wdl.draft3.transforms.ast2wdlom
+
+import cats.syntax.either._
+import cats.syntax.apply._
+import common.validation.ErrorOr.ErrorOr
+import wdl.draft3.parser.WdlParser.Ast
+import wdl.model.draft3.elements.RuntimeAttributeElement
+
+object AstToRuntimeAttributeElement {
+  def convert(ast: Ast): ErrorOr[RuntimeAttributeElement] = {
```

Comment:
I think you should be able to repurpose the existing `AstNodeToKvPair.convert`, followed by a `map` into a `RuntimeAttributeEntryElement`

Reply:
Would it make more sense for RuntimeAttributeEntryElement to contain a kvPair and have this convert delegate to AstNodeToKvPair.convert, then construct a RuntimeAttributeEntryElement with the returned value?

Reply:
So that would make building them easier but the indirection might be awkward when we actually have to read from them. One other option it to use `KvPair` directly in the `RuntimeSection`, eg `case class RuntimeSection(values: Seq[KvPair])`, since I think they really KvPair really is exactly the same as `RuntimeAttributeElement `